app/services/phone_number_capabilities.rb

```diff
@@ -3,10 +3,11 @@ class PhoneNumberCapabilities
     Rails.root.join('config', 'country_dialing_codes.yml'),
   ).freeze
 
-  attr_reader :phone
+  attr_reader :phone, :phone_confirmed
 
-  def initialize(phone)
+  def initialize(phone, phone_confirmed)
```

Comment:
WDYT of making `phone_confirmed` a keyword argument, so it's clearer in callsites what the hardcoded `true` / `false` are for?

```suggestion
  def initialize(phone, phone_confirmed:)
```

Reply:
👍🏼